src/pages/Introduction/Introduction.js

```diff
@@ -0,0 +1,130 @@
+// @flow strict
+
+import React, { PureComponent } from 'react'
+
+import { StartLayout } from 'layouts'
+import { t } from 'ttag'
+import ViewSlider from 'react-view-slider'
+
+import {
+  JLink,
+  JRaisedButton,
+} from 'components/base'
+
+import {
+  setIntroductionValue,
+} from 'utils/introduction'
+
+import { Slide } from './components/Slide'
+import { SliderButton } from './components/SliderButton'
+
+import IntroductionStyle from './introduction.m.scss'
+
+export type Props = {|
+|}
+
+type ComponentState = {|
+  +activeId: number,
+|}
+
+const sliders = [
+  {
+    img: '/assets/feature/jwallet.png',
+    animationName: 'jwallet',
+    title: t`Jwallet`,
+    descr: t`A simple, fast and secure mobile wallet 
+      <br> for Ethereum and all ERC-20 tokens.`,
+  },
+  {
+    img: '/assets/feature/store.png',
+    animationName: 'store',
+    title: t`Store`,
+    descr: t`Create and manage several wallets for different goals or just simply 
+      <br> import your existing Ethereum and ERC-20 wallet.`,
+  },
+  {
+    img: '/assets/feature/protect.png',
+    animationName: 'protect',
+    title: t`Protect`,
+    descr: t`All sensitive data never leaves your device and your private keys 
+      <br> are never shared with anyone, including us.`,
+  },
+  {
+    img: '/assets/feature/send_receive.png',
+    animationName: 'send_receive',
+    title: t`Send & Receive`,
+    descr: t`All sensitive data never leaves your device and your private keys 
+      <br> are never shared with anyone, including us.`,
+  },
+  {
+    img: '/assets/feature/manage.png',
+    animationName: 'manage',
+    title: t`Manage`,
+    descr: t`Enjoy complete control over your digital assets. 
+      <br> Manage ETH and all ERC-20 tokens.`,
+  },
+]
+
+const renderView = ({
```

Comment:
So, you can actually put slides data into separate file, e.g. `data.js`, and move this function inside `ViewSlider` component, eliminating the function purpose, yes?